src/client/app/components/UIOptionsComponent.tsx

```diff
@@ -5,39 +5,39 @@
 import * as React from 'react';
 import sliderWithoutTooltips, { createSliderWithTooltip } from 'rc-slider';
 import * as moment from 'moment';
-import { Button, ButtonGroup } from 'reactstrap';
+import { Button, ButtonGroup, Dropdown, DropdownToggle, DropdownMenu, DropdownItem } from 'reactstrap';
```

Comment:
Could you remove TimeInterval from the line below? Just a code maintenance thing.